src/scss/initial/_headings.scss

```diff
@@ -30,7 +30,8 @@ h2 {
 }
 
 h2,
-h3 {
+h3,
+.fancy-heading {
```

Comment:
@SondraE Is there a logic to when we want gradient backgrounds on the main page title? Should we always have it? Or always on pages without a hero image? Or is it only for specific pages?